packages/framework/src/Command/TranslationReplaceSourceCommand.php

```diff
@@ -132,7 +132,9 @@ private function extractReplacementsFromPoFile(SplFileInfo $file)
         $translationSourceReplacements = [];
         $item = $defaults;
 
-        while ($line = fgets($stream)) {
+        $line = fgets($stream);
```

Comment:
takova pocitovka, ale vsechny tyhle while bych spis hodil jako:

```php
while (true) {
    $line = fgets($stream);

   if ($line === false) {
      break;
  }
}
```

ale klidne na to pec